src/Authoring/AuthoringSample/AuthoringSample.csproj

```diff
@@ -1,28 +1,22 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
-    <Platforms>x64;x86</Platforms>
-    <LangVersion>preview</LangVersion>
-    <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
-    <CsWinRTComponent>true</CsWinRTComponent>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <CompilerVisibleProperty Include="AssemblyName" />
-    <CompilerVisibleProperty Include="AssemblyVersion" />
-    <CompilerVisibleProperty Include="CsWinRTComponent" />
-    <CompilerVisibleProperty Include="CsWinRTEnableLogging" />
-    <CompilerVisibleProperty Include="GeneratedFilesDir" />
-    <CompilerVisibleProperty Include="CsWinRTExe" />
-    
-    <ProjectReference Include="..\..\Projections\Windows\Windows.csproj" />
-    <ProjectReference Include="..\WinRT.SourceGenerator\WinRT.SourceGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
-    <ProjectReference Include="..\..\cswinrt\cswinrt.vcxproj" />
-    <ProjectReference Include="..\..\WinRT.Runtime\WinRT.Runtime.csproj" />
-  </ItemGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
should this be moved to the Samples folder?

Reply:
I probably want to rename this to be a test, this component is more for test purposes than to serve as a sample.  I will address that in a future PR.